### PR TITLE
[CALCITE-6213] The default behavior of NullCollation in Presto is LAST

### PR DESCRIPTION
**Pull request by YiwenWu**
This PR fix the NullCollation of Presto dialect, changed from "LOW" to "LAST"

https://issues.apache.org/jira/projects/CALCITE/issues/CALCITE-6213

### CODE REVIEW

File: core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java [hunk 1]
```diff
@@ -7704,6 +7703,101 @@ private void checkLiteral2(String expression, String expected) {
...
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6213">[CALCITE-6213]
+   * The default behavior of NullCollation in Presto is LAST </a>.
+   */
+  @Test void testNullCollation() {
+    final String query = "select * from \"product\" order by \"brand_name\"";
+    final String expected = "SELECT *\n"
+        + "FROM \"foodmart\".\"product\"\n"
+        + "ORDER BY \"brand_name\"";
+    final String sparkExpected = "SELECT *\n"
```
[the remark this answers is not in the document]
Comment (YiwenWu) [reply core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java:7715]: SparkSQL and Presto are commonly used in SQL-on-Hadoop frameworks. Previously, Presto was set to the value "LOW," which is consistent with Spark. This test demonstrates the comparative effect of setting null  collation to "LAST."

**Comment by XuQianJin-Stars** [comment]
hi @YiwenWu Thanks for this work，Please improve the PR description and add isuse link.

**Comment by YiwenWu** [comment]
> hi @YiwenWu Thanks for this work，Please improve the PR description and add isuse link.

@XuQianJin-Stars Thank you for the code review,  I added the description and link.

**Review by JiajunBernoulli — APPROVED** [review]

**Comment by JiajunBernoulli** [comment]
LGTM, thanks for your PR @YiwenWu

**Review by macroguo-ghy — APPROVED** [review]
+1, @YiwenWu Thanks for your contribution!

**Pull request merged**